apps/e2e-test/src/playwright-support/generic/verification.ts

```diff
@@ -1,48 +1,89 @@
-import { expect } from '@playwright/test';
 import { logger } from '../../logger';
+import assert from 'assertion';
 
-export const verifyTextMatch = (actText: string, expPattern: string, negate = false): void => {
+export const verifyMatch = (
+  actual: string,
+  expPattern: string,
+  options?: { negate?: boolean; message?: string }
```

Comment:
We'd better not use an argument to change the behavior here, instead, we should have something like `verifyNotMatch`.  Using argument makes it more complicated to write and understand. This applys to below verification too.

Reply:
We could use this argument in an internal function that serves public functions, but for public APIs, I suggest using simple functions without arguments changing it's intention.

Reply:
Fixed.